delivery_transsmart/models/product_template.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    package = fields.Boolean()
+    service_level_time_id = fields.Many2one(
+        'service.level.time',
+        string='Service Level Time',
+    )
+    code = fields.Char()
+    nr = fields.Integer('Identifier')
```

Comment:
maybe rename these kind of fields to transsmart_id? The name nr is not very descriptive...

Reply:
@NL66278 I purposefully let them have the same names as in the Transsmart's API so that I can avoid adding and maintaining extra complexity of mapping around local fields with remote ones. So, the synchronization is simpler as well. These fields are not shown or modifiable by the end user.